tgui/packages/tgui/styles/layouts/NtosHeader.scss

```diff
@@ -18,3 +18,23 @@
   margin-bottom: -0.5em;
   vertical-align: middle;
 }
+
+.alertBlink {
+  color: #FFFFFF;
+  animation: alertBlinkAnimation 1s infinite;
```

Comment:
id do this only 3 times instead of infinite, its not THAT important, up to you tho